core/src/test/java/org/springframework/security/authorization/AuthorityAuthorizationManagerTests.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.authorization;
+
+import java.util.function.Supplier;
+
+import org.junit.Test;
+
+import org.springframework.security.authentication.TestingAuthenticationToken;
+import org.springframework.security.core.Authentication;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+/**
+ * Tests for {@link AuthorityAuthorizationManager}.
+ *
+ * @author Evgeniy Cheban
+ */
+public class AuthorityAuthorizationManagerTests {
+
+	@Test
+	public void testWhenRoleNull() {
```

Comment:
To better align with the team standard, would you mind renaming these methods to `methodUnderTestWhenConditionThenResult`?

For example, this method could be instead `hasRoleWhenNullThenException`.